### ISSUE TITLE
"Clear" button for Query Editor

### ISSUE THREAD

**Issue by harshithpabbati** [graphql/graphiql#1388]
Clearing queries in the Query Editor could make use of a "Clear" button in the user interface of the Query Editor to make it easier to clear all content present in it quickly, just as with the "Copy" button (with a possible keyboard shortcut - maybe `Shift-Ctrl-L` or similar?) 

![Screenshot 2020-03-02 at 8 58 29 PM](https://user-images.githubusercontent.com/43822585/75690304-9a4a5a00-5cc8-11ea-956e-2c4b377d3de7.png)

**Comment by harshithpabbati** [comment]
I would like to work on this

**Comment by acao** [comment]
we are switching out state management, so a clear command will exist soon!

**Comment by beerose** [comment]
Hi @acao! Any ETA on this redesign?

**Comment by acao** [comment]
it is well underway! see pinned issues

**Comment by beerose** [comment]
Yeah, I did. Also checked the milestone, but there's no due date, so I wanted to ask if you maybe have some rough estimation on the 2.0 beta.
